inspec/a2-api-integration/controls/iam_v2.rb

```diff
@@ -427,63 +428,207 @@
+      end
+
+      it "user gets a 404 when the user does not exist" do
+        resp = automate_api_request("/apis/iam/v2beta/users/some_wrong_id")
+        expect(resp.http_status).to eq 404
+      end
+    end
+
+    describe "PUT /apis/iam/v2beta/self/:id" do
+      it "user can update their own display name" do
+          updatedName =  "i updated my own name"
+          resp = automate_api_request("/apis/iam/v2beta/self/#{CUSTOM_USER_ID}",
+            http_method: 'PUT',
+            request_body: {
+              name: updatedName
+            }.to_json
+          )
+          expect(resp.http_status).to eq 200
+          expect(resp.parsed_response_body[:user][:name]).to eq updatedName
+        end
+
+        it "user gets a 400 if their password is wrong" do
+          updatedName =  "i updated my own name"
+          resp = automate_api_request("/apis/iam/v2beta/self/#{CUSTOM_USER_ID}",
+            http_method: 'PUT',
+            request_body: {
+              name: updatedName,
+              password: "newpassword",
+              previous_password: "wrongagain"
+            }.to_json
+          )
+          expect(resp.http_status).to eq 400
+        end
+
+        # TODO (tc): right now, it returns 400 for a non-existent user because
+        # it checks for the password first, and if it doesn't match for a user
+        # that doesn't exist (always the case), it returns a 400.
+        it "returns 400 for a non-existent user" do
+          updatedName =  "i updated my own name"
+          resp = automate_api_request("/apis/iam/v2beta/self/some_wrong_id",
+            http_method: 'PUT',
+            request_body: {
+              name: updatedName,
+              password: "newpassword",
+              previous_password: "chefautomate"
+            }.to_json
+          )
+          expect(resp.http_status).to eq 400
+        end
+
+        it "user can update their own display name and password" do
+          updatedName =  "i updated my own name"
+          resp = automate_api_request("/apis/iam/v2beta/self/#{CUSTOM_USER_ID}",
+            http_method: 'PUT',
+            request_body: {
+              name: updatedName,
+              password: "newpassword",
+              previous_password: "chefautomate"
+            }.to_json
+          )
+          expect(resp.http_status).to eq 200
+          expect(resp.parsed_response_body[:user][:name]).to eq updatedName
+        end
+      end
+
+      describe "DELETE /iam/v2beta/users/:id" do
+        it "deletes the user if it exists" do
+          resp = automate_api_request("/apis/iam/v2beta/users/#{CUSTOM_USER_ID}", http_method: 'DELETE')
+          expect(resp.http_status).to eq 200
+        end
+
+        it "user gets a 404 when the user does not exist" do
+          resp = automate_api_request("/apis/iam/v2beta/users/some_wrong_id", http_method: 'DELETE')
+          expect(resp.http_status).to eq 404
+        end
+      end
+
+      describe "PUT /iam/v2beta/users/:id" do
```

Comment:
Since these test cases are no longer about the user updating itself, could we 

1) Put them in a describe block that clarifies that 
2) Change the `updatedName` text (which makes the reader think we're still in the `self` block)?